examples/complete/test.tfvars

```diff
@@ -0,0 +1,91 @@
+project_name = "codebuild-osahon"

```

Comment:
This should be a descriptive name, preferably with a random suffix to avoid collisions if more than one person were running the tests.

Reply:
I have renamed the project in the test.tfvars file to codebuild_project_1.